src/Workspaces/CoreTest/UtilityTest/DocumentationCommentIdTests.cs

```diff
@@ -351,5 +351,21 @@ class InnerType<B, C>
             CheckReferenceId("System.Int32[]", compilation.CreateArrayTypeSymbol(intType), compilation);
             CheckReferenceId("System.Int32*", compilation.CreatePointerTypeSymbol(intType), compilation);
         }
+
+        [Fact, WorkItem(65396, "https://github.com/dotnet/roslyn/issues/65396")]
+        public void TestInvalidTypeParameterId()
```

Comment:
>TestInvalidTypeParameterId [](http://example.com/codeflow?start=20&length=26)

I think we need to add similar test for VB

Reply:
Thanks. I haven't found VB tests for `DocumentatoinCommentId`, so I added it in the same class: 64443c472934146349ab021c6b3199efeed7d4c1